From 9fb05323291ccdfbf19ac0d9428e366d6023b408 Mon Sep 17 00:00:00 2001
From: Karthick Sundarrajan <karthick.sundarrajan@intel.com>
Date: Fri, 3 Apr 2020 10:23:41 -0700
Subject: [PATCH] Disable LLMNR (port 5355)

As part of OS hardening process, the port has to be
disabled.

Signed-off-by: Karthick Sundarrajan <karthick.sundarrajan@intel.com>
---
 src/resolve/resolved.conf.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/resolve/resolved.conf.in b/src/resolve/resolved.conf.in
index 6898c78..a9125fd 100644
--- a/src/resolve/resolved.conf.in
+++ b/src/resolve/resolved.conf.in
@@ -15,7 +15,7 @@
 #DNS=
 #FallbackDNS=@DNS_SERVERS@
 #Domains=
-#LLMNR=yes
+LLMNR=no
 #MulticastDNS=yes
 #DNSSEC=@DEFAULT_DNSSEC_MODE@
 #DNSOverTLS=@DEFAULT_DNS_OVER_TLS_MODE@
